### PR TITLE
SOC-1109 Add Redirect URL param to tracking data

### PR DESCRIPTION
**Pull request by bkoval**
@kvas-damian @rafalkalinski 
https://wikia-inc.atlassian.net/browse/SOC-1109

### CODE REVIEW

File: front/scripts/mercury/modules/Trackers/Internal.ts [hunk 1]
```diff
@@ -46,17 +47,23 @@ module Mercury.Modules.Trackers {
...
+					lc: mercury.wiki.language.user,
+					u: 0,
+					s: 'mercury',
+					beacon: '',
+					cb: ~~(Math.random() * 99999)
+				},
+				redirectUrl: string = Mercury.Utils.getQueryParam('redirect');
+
+			if (redirectUrl) {
+				config.redirectUrl = redirectUrl;
```
Comment (kvas-damian) [inline front/scripts/mercury/modules/Trackers/Internal.ts]: should it be called redirectUrl?

File: front/scripts/auth/tracking.ts [hunk 2]
```diff
@@ -28,6 +30,8 @@
 		dimensions[5] = 'anon';
 		// IsCorporatePage
 		dimensions[15] = 'No';
+		// newAuthEntryPage
+		dimensions[10] = Mercury.Utils.getQueryParam('redirect');
```
Comment (rogatty) [inline front/scripts/auth/tracking.ts]: `M.getQueryParam` should work too, right?

File: front/scripts/mercury/modules/Trackers/Internal.ts [hunk 3]
```diff
@@ -46,17 +48,23 @@ module Mercury.Modules.Trackers {
...
+			}
+
+			return config;
```
Comment (kvas-damian) [inline front/scripts/mercury/modules/Trackers/Internal.ts]: I don't like place where we added this change. Maybe we could create a way to pass additional params via config when instantiating tracker (in a constructor) or optional field in tracker method?

**Comment by kvas-damian** [comment]
LGTM

**Comment by rafalkalinski** [comment]
LGTM

**Pull request merged**